Source/WebCore/rendering/RenderListItem.cpp

```diff
@@ -82,7 +82,7 @@ RenderStyle RenderListItem::computeMarkerStyle() const
     markerStyle.fontCascade().update(&document().fontSelector());
     markerStyle.setUnicodeBidi(UnicodeBidi::Isolate);
     markerStyle.setWhiteSpaceCollapse(WhiteSpaceCollapse::Preserve);
-    markerStyle.setTextWrap(TextWrap::NoWrap);
+    markerStyle.setTextWrapMode(TextWrapMode::NoWrap);
```

Comment:
Just confirming that we do not need to also set `text-wrap-style` here?

Reply:
Same here